js-miniapp-bridge/src/common-bridge.ts

```diff
@@ -99,6 +100,20 @@ export class MiniAppBridge {
       );
     });
   }
+
+  /**
+   * Associating showInterstitialAd function to MiniAppBridge object
+   */
+  showInterstitialAd() {
+    return new Promise<InterstitialAdResponse>((resolve, reject) => {
+      return this.executor.exec(
+        'showAd',
+        { adType: AdTypes.INTERSTITIAL },
+        adResponse => resolve(JSON.parse(adResponse) as InterstitialAdResponse),
```

Comment:
Could you tell me how the response of `adResponse` looks like? The example would be good.

Reply:
`const jsonAdresponse = '{ "adType": 1 }';`
js-miniapp-bridge/test/test.spec.ts :line 80